src/main/java/seedu/condonery/logic/parser/CondoneryParser.java

```diff
@@ -25,7 +26,8 @@ public class CondoneryParser {
     /**
      * Used for initial separation of command word and args.
      */
-    private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static final Pattern BASIC_COMMAND_FORMAT = Pattern
+            .compile("(?<commandWord>\\S+\\s-[pc]{1})(?<arguments>.*)");
```

Comment:
hm i'm wondering if `-[pc]{1}` should be optional, since we might have commands such as `help` that would not match the regex.

Reply:
Ah right, thanks! I forgot that